webserver/sf-perftest-metrics/app.yaml

```diff
@@ -27,7 +27,3 @@ objectives:
   - name: cost
     requests:
       selector: app=voting-app
-stormForger:
```

Comment:
@jgustie Please confirm that this section being dropped out by `fix` is intended.

Reply:
Correct, the `stormForger` section is no longer required.

Reply:
Cool. But when I try to stormforge generate experiment, it fails with some oauth issue. Is it trying to reach out to an external service?

Reply:
Thanks for the context. I was told to drop all the app.yamls in this repo in favor of generated experiment files, but looks like the Application CRD is here to stay? At least for cases like these?

Reply:
No, the plan is still to drop app.yaml; the `stormforge gen exp` command is hidden and only remains for a transitional period. The CLI alternative is to use `stormforge run` to interactively build an experiment.

Reply:
I am not sure when `vhs-sf-perftest/README.md` will be updated, but it talks about creating your secrets manually with `STORMFORGER_JWT` being part of them.